spk/homeassistant/Makefile

```diff
@@ -83,6 +83,11 @@ ENV += PYO3_CROSS_INCLUDE_DIR=$(STAGING_INSTALL_PREFIX)/include/
 # [cryptography]
 DEPENDS += cross/cryptography_38
 
+# [uamqp]
+# ignore deprecation warnings when building with openssl 3
+# "cc1: all warnings being treated as errors"
+WHEELS_CFLAGS = [uamqp] -Wno-deprecated-declarations
+
```

Comment:
@th0ma7 unfortunaltely this didn't work 
A solution for uamqp is documented here: https://github.com/Azure/azure-uamqp-python/issues/276#issuecomment-924890556
but it requires to patch the source code (and the patch file isn't available anymore)
I will try to create cross/uampq to build this wheel.

Reply:
It is not possible to build uamqp with DSM 6 toolchains.
Therefore homeassistant 2023.1.7 must  be built with openssl 1.1.1.

Reply:
I may have found it from the Debian repository... 
https://salsa.debian.org/python-team/packages/azure-uamqp-python/-/blob/debian/1.5.0-1/debian/patches/OpenSSL3.patch